lib/srv/ctx.go

```diff
@@ -393,6 +393,12 @@ type ServerContext struct {
 	killShellr *os.File
 	killShellw *os.File
 
+	// multiWriter is used by to stream non-interactive session output
+	// to audit log.
```

Comment:
audit log or session recording? would this be more accurate?
```suggestion
	// multiWriter is used to record non-interactive session output
```